src/protocols/router/hpr_router_relay.erl

```diff
@@ -61,14 +62,13 @@ init([GatewayStream, RouterStream]) ->
     {noreply, #state{}, {continue, relay}}
     | {stop, normal, #state{}}
     | {stop, {error, any()}, #state{}}.
-handle_continue(relay, State) ->
-    case grpc_client:rcv(State#state.router_stream) of
+handle_continue(relay, #state{router_stream = RouterStream, pubkey_bin = PubKeyBin} = State) ->
+    case grpc_client:rcv(RouterStream) of
         {data, Map} ->
-            PacketDown = hpr_packet_down:to_record(Map),
-
-            lager:debug("sending router downlink.  pid: ~p", [State#state.gateway_stream]),
-
-            ok = hpr_packet_service:packet_down(State#state.gateway_stream, PacketDown),
+            lager:debug("sending router downlink to : ~p", [hpr_utils:gateway_name(PubKeyBin)]),
```

Comment:
gateway-name is metadata so it's queryable.
We could add it when the relay is initialized, similar to the gwmp worker.